frontend/unchalli-falls/unchalli.css

```
@@ -0,0 +1,1714 @@
+.unchalli-hero-image {
+  position: absolute;
+  inset: 0;
+  background-image:
+    url("../assets/images/unchalli-cascade.jpg");
+  background-size: cover;
+  background-position: center;
+  transform: scale(1.04);
+  animation: ufHeroZoom 16s ease-in-out infinite alternate;
+}
+
+@keyframes ufHeroZoom {
```

Stylelint (17.14.0)

[error] Expected keyframe name "ufHeroZoom" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)